assignment_3/A3-ShapeMatching-CodeHandout/my_bit_ga.m

```diff
@@ -0,0 +1,43 @@
+function output = my_bit_ga(nacafoil, total_eval, verbose)
+
+% Algorithm Parameters
+popSize = 50;
+nGenes  = 32;
+maxGen = total_eval/popSize;
+sp = 2; % selection pressure
+mutProb = 1/nGenes; % probability for an individual to mutate
+crossProb = 0.99; % probability for crossover
+bestFit = zeros([maxGen, 1]);
+medianFit = zeros([maxGen, 1]);
+    
+% Create an individual (consisting of 32 y values between -0.5 and 0.5)
+pop = randi(2,popSize, 32)-1;
```

Comment:
Don't get why 1 is subtracted here.

Reply:
Because `randi` will give matrix filled with `1` & `2`. We want it to be `0` & `1`.

Reply:
Clear